File: analysisapi/presentation/dto/post_diary_request.py
"""日記のベクトルDBに登録するためのリクエストDTOです。"""

from pydantic import BaseModel


class PostDiaryRequest(BaseModel):
    """日記のベクトルDBに登録するためのリクエストDTOです。"""

    id: int
    diary_text: str

File: backend/web/src/main/java/com/memoblend/web/controller/dto/diary/PostDiaryRequest.java
package com.memoblend.web.controller.dto.diary;

import java.time.LocalDate;
import lombok.Data;

/**
 * 日記の投稿リクエストクラスです。
 */
@Data
public class PostDiaryRequest {
  private final long userId;
  private final String title;
  private final String content;
  private final LocalDate createdDate;
}
